source/main/modules/window-manager/index.ts

```diff
@@ -207,18 +213,50 @@ export default class WindowManager {
   /**
    * Shuts down the window manager and performs final operations
    */
-  shutdown (): void {
+  shutdown(): void {
     this._persistWindowPositions()
   }
 
   /**
    * Listens to events on the main window
    */
-  private _hookMainWindow (): void {
+  private _hookMainWindow(): void {
     if (this._mainWindow === null) {
       return
     }
 
+    this._mainWindow.on('show', async () => {
+      // issue : Add tray to MacOS #4
+      // If Zettlr is running AND the Show app in the notification area is true, 
+      // the tray icon will be visible (regardless if Zettlr window(s) are visible or not).
+      // Zettlr can take a while a load. Only show the tray when the Zettlr window is ready.
+      // Do not add the actions (event handlers) to tray click and tray menu. See separate issue.
+      if (process.platform == 'darwin') {
+        if (tray == null) {
+          let basepath = app.getAppPath()
+          let image = await nativeImage.createThumbnailFromPath(basepath + '/source/main/assets/icons/black_white_128.png', { width: 16, height: 16 })
```

Comment:
I don't think the path `/source/` will work when Zettlr is [packaged](https://github.com/UofA-SPI21-team7/zettlr-assignment/wiki/Zettlr-creating-packages). See #18 for my proposed solution on Windows.

Reply:
It will fix in new commit